<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckoutDialog</class>
 <widget class="QDialog" name="CheckoutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>602</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>761</width>
    <height>602</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>761</width>
    <height>602</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Thanh toán</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../resources.qrc">
    <normaloff>:/images/icon/note.png</normaloff>:/images/icon/note.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#CheckoutDialog {
	background-color:#f5f5fa;
}</string>
  </property>
  <widget class="QLabel" name="label_50">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 28px;
font-weight: bold;
color: #f27d85;</string>
   </property>
   <property name="text">
    <string>Thanh toán</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="checkoutWrapper" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>721</width>
     <height>535</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#checkoutWrapper {
	background-color: white;
	border-radius: 20px;
}</string>
   </property>
   <widget class="QWidget" name="formWidget_4" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>7</y>
      <width>681</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#label_51 {
	font-size: 16px;
	font-weight: bold;
	color: #333;
}</string>
    </property>
    <layout class="QFormLayout" name="formLayout_9">
     <property name="labelAlignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
     <property name="horizontalSpacing">
      <number>12</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_51">
       <property name="styleSheet">
        <string notr="true">color: #333;</string>
       </property>
       <property name="text">
        <string>Địa chỉ giao hàng:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="address">
       <property name="styleSheet">
        <string notr="true">padding: 4px 8px;
font-size: 16px;
border: 1px solid #777;
border-radius: 10px;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QScrollArea" name="checkoutScroll">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>681</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QScrollArea {
	border-radius: 15px;
	background-color: white;
}

QScrollBar:vertical {              
	border: 1px solid #999999;
	border-radius: 5px;
    background: transparent;
    width: 12px;    
    margin: 0px 0px 0px 0px;
}

QScrollBar::handle:vertical {
    background: #f27d85;
	border-radius: 5px;
}

QScrollBar::add-line:vertical {
    height: 0px;
    subcontrol-position: bottom;
    subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical {
    height: 0 px;
    subcontrol-position: top;
    subcontrol-origin: margin;
}</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>681</width>
       <height>341</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#scrollAreaWidgetContents {   
    background-color: white;

}
</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="checkoutBtn">
    <property name="geometry">
     <rect>
      <x>577</x>
      <y>482</y>
      <width>121</width>
      <height>35</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 15px;
	color: white;
	font-size: 16px;
	font-weight: bold;
	background-color: #f27d85;
	border: none;
}
</string>
    </property>
    <property name="text">
     <string>Thanh toán</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_28" native="true">
    <property name="geometry">
     <rect>
      <x>242</x>
      <y>477</y>
      <width>331</width>
      <height>46</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 16px;
font-weight: bold;</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_27" stretch="1,0">
     <item>
      <widget class="QLabel" name="label_63">
       <property name="styleSheet">
        <string notr="true">color: #333;
font-size: 18px;</string>
       </property>
       <property name="text">
        <string>Tổng tiền:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="totalCost">
       <property name="styleSheet">
        <string notr="true">color: #f27d85;
font-size: 20px;</string>
       </property>
       <property name="text">
        <string>100.000đ</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="payOption">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>484</y>
      <width>222</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
	padding: 4px 8px;
	font-size: 16px;
	color: #333;
	background-color: white;
	border: 1px solid #777;
	border-radius: 15px;
}

QComboBox QAbstractItemView {
    border: 1px solid darkgray;
	padding: 4px 8px;
	border-radius: 10px;
}</string>
    </property>
    <item>
     <property name="text">
      <string>Thanh toán khi nhận hàng</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sử dụng Ví Bily</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="formWidget_5" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>423</y>
      <width>681</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font-size: 16px;
	font-weight: bold;
	color: #333;
}</string>
    </property>
    <layout class="QFormLayout" name="formLayout_10">
     <property name="labelAlignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
     <property name="horizontalSpacing">
      <number>12</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_54">
       <property name="styleSheet">
        <string notr="true">color: #333;</string>
       </property>
       <property name="text">
        <string>Ghi chú giao hàng:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="note">
       <property name="styleSheet">
        <string notr="true">padding: 4px 8px;
font-size: 16px;
border: 1px solid #777;
border-radius: 10px;</string>
       </property>
       <property name="placeholderText">
        <string>Nhập ghi chú giao hàng...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>681</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top-width: 2px;
border-top-color: #f27d85;
border-top-style: solid;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>681</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top-width: 2px;
border-top-color: #f27d85;
border-top-style: solid;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
